recipes/libpqxx/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,28 @@
+#include "pqxx/pqxx"
+
+#include <cstdlib>
+#include <iostream>
+
+
+int main()
+{
+    std::cout << "libpqxx version: " << PQXX_VERSION << std::endl;
+
+    try {
+        pqxx::connection conn("hostaddr='.conan' port=5432 dbname=conan user=conan connect_timeout=2");
+
+        pqxx::work tx(conn);
+
+        pqxx::row row = tx.exec1("SELECT 1");
+
+        tx.commit();
+
+        std::cout << row[0].as<int>() << "\n";
+    } catch (const pqxx::broken_connection &ex) {
+        std::cout << "error thrown: " << ex.what() << "\n";
+
+        return EXIT_SUCCESS;
+    }
+
+    return EXIT_FAILURE;
```

Comment:
```suggestion
    return EXIT_SUCCESS;
```

If the connection works for some reason (which is more expected), the test will fail.

Reply:
The libpq documentation states that the `hostaddr`parameter can only be specified in IP address format.

https://www.postgresql.org/docs/11/libpq-connect.html;

> Numeric IP address of host to connect to. This should be in the standard IPv4 address format, e.g., 172.28.40.9. If your machine supports IPv6, you can also use those addresses. TCP/IP communication is always used when a nonempty string is specified for this parameter.

`hostaddr='.conan'` is treated as an invalid parameter.
I expect the connection to always fail.

Reply:
Okay, so we are good. Thanks for clarifying!